flow-server/src/test/java/com/vaadin/flow/server/communication/LongPollingCacheFilterTest.java

```diff
@@ -81,6 +81,19 @@ public void filter_notLongPollingTransport_continueWithCurrentMessage() {
         verifyMessageIsNotCached();
     }
 
+    @Test
+    public void filter_syncIdCheckDisabled_continueWithCurrentMessage() {
+        setTransport(AtmosphereResource.TRANSPORT.LONG_POLLING);
+        setSeenServerSyncIdHeader(-1);
+        BroadcastAction action = filter.filter("broadcasterId", resource,
+                originalMessage, message);
+        Assert.assertEquals(ACTION.CONTINUE, action.action());
+        Assert.assertSame(
+                "Message should not be altered by filter if server sync id header is missing",
```

Comment:
A minor change on the assertion message
```suggestion
                "Message should not be altered by filter if syncId check is disabled",
```

Reply:
Fixed